config/sync/core.extension.yml

```
@@ -216,6 +216,8 @@ module:
 theme:
   seven: 0
   vagovadmin: 0
+  claro: 0
+  vagovclaro: 1000
```

If the D.O patch solved the requirements issue.  Let's set the weight back to 0.
```suggestion
  vagovclaro: 0
```